### ISSUE TITLE
Apple Music should not be tagged as "Downloadable"

### ISSUE THREAD

**Issue by elliotchance** [jgchk/better-rym#148]
Apple Music is not the same as the iTunes Store (which I believe is going away soon). Tracks on Apple Music are encrypted and can only be played on an apple device that's authorized against that account. So it's not fair to day these are Downloadable.

Also see https://rateyourmusic.com/admin/corq/?req_id=10106042

**Comment by sc113** [comment]
This attribute is called "Downloadable" and not "Downloadable and unencrypted" or "Downloadable and free to redistribute". It is just about downloading a complete file in audio format, without any additional conditions.

You should probably discuss it in the forum and get a consensus if you don't agree with it. For now, we should be guided by https://rateyourmusic.com/list/fourths/submission-quick-ref-sheet/

**Comment by elliotchance** [comment]
@sc113 there is a condition though, that it is available on the iTunes Store (which has different catalog to Apple Music), if you look at the quick sheet you linked:

```
Apple Music/iTunes: Lossless Digital Streaming, Downloadable*

*Not all releases available for download.
```

So it's already decided that being on Apple Music doesn't mean it has to be downloadable. Conversely, Spotify does allow you to download cached (encrypted files) in exactly the same way as Apple Music but it is not considered Downloadable:

```
Spotify: Digital File Streaming
```

The real question is whether the _default_ should be yes or no. Apple Music usage far outweighs iTunes Store and, like I said the iTunes Store is actually going away at some point (see below), so it doesn't make sense to make the default assuming that person has linked the purchasable iTunes album instead of the streamable Apple Music version.

> The shutdown shouldn’t shock anyone. In June [2019], Apple announced that it was shutting the app down for good.  The company said at the time that the app was simply too cluttered.  For years, users have been complaining about iTunes ‘bloatware’ as piles of different media categories kept fattening the platform. - https://www.digitalmusicnews.com/2019/10/08/itunes-apple-shut-down-itunes/

**Comment by sc113** [comment]
I think at this point it would be more relevant to check for the _"Also available in the iTunes Store"_ element on the page and add downloadable if there is one.

**Comment by jgchk** [comment]
implemented in faa9e8af962e7687143f6132fc851a1dc80701a9

default -> digital file, streaming
page has "_Also available in the iTunes Store_" -> lossless digital, downloadable, streaming

**Comment by elliotchance** [comment]
Thanks @jgchk !
